plugin/storage/factory_config.go

```diff
@@ -69,11 +81,41 @@ func FactoryConfigFromEnvAndCLI(args []string, log io.Writer) FactoryConfig {
 	if depStorageType == "" {
 		depStorageType = spanWriterTypes[0]
 	}
+	ratioEnvVariable := os.Getenv(DownsamplingRatio)
```

Comment:
Can we read this via normal CLI options? spanStorageType is a special case because it informs which factory implementation to choose before parsing CLI flags. But downsampler params can be read in InitFromViper()